pipeline/json.go

```diff
@@ -447,6 +448,9 @@ func unmarshalStats(data []byte, parents []Node, typ TypeOf) (Node, error) {
 	}
 	child := nodeParent.Stats(0)
 	err := json.Unmarshal(data, child)
+	if child.Interval == 0 {
+		return nil, errors.New("zero is an invalid stats interval")
```

Comment:
This code is for serializing kapacitor pipelines, isn't it? Doesn't this check belong in `stats.go`?

Reply:
I added one to stats.go